logger/logger.go

```diff
@@ -20,6 +21,10 @@ var (
 	// LogWriter is where the logs go. By default this empty buffer blackholes
 	// logs. It must be an io.Writer such as os.Stdout
 	Writer io.Writer = ioutil.Discard
+
+	// regex to format grpc call logs
+	// "/service.Service/Endpt" -> "Endpt"
+	FormatServiceEndpoint = regexp.MustCompile(`\/([A-Za-z])\w+\.([A-Za-z])\w+\/`)
```

Comment:
no reason, you guys suggested it go to the package level, I moved it to the package level with the other vars... I can make it a constant